app/controllers/external_resources_controller.rb

```diff
@@ -22,7 +22,7 @@ def create
 
     respond_to do |format|
       if @external_resource.save
-        format.html { redirect_to external_resources_path, notice: 'External resource was successfully created.' }
+        format.html { redirect_to @admin_status ? external_resources_path : community_resources_path, notice: "External resource was successfully submitted.#{ " We'll review." unless @admin_status }" }
```

Comment:
Here's an example of why having @admin_status can be so useful. After an admin creates a resource, they'd go to the admin-index where there's a condensed index view w edit links, whereas a non-authenticated user should get taken back to the pretty list. :)